games/static/games/proto.js

```diff
@@ -274,13 +271,21 @@ class Board {
 
     moveDefender(direction){
         let defender = this.pickDefenderUp()
+        let nextMove = this.calculateNextDefenderMove(direction)
+        this.defenderMove(nextMove)
+        this.putDown(defender)
+    }
+    calculateNextDefenderMove(direction) {
         if (this.defenderCanMoveLeft(direction)) {
-            this.defenderMoveLeft()            
+            return directionLeft            
```

Comment:
rename constant to be aligned with google style guide:
https://google.github.io/styleguide/jsguide.html#naming-constant-names